File: python/easy/create_target_array_in_given_order.py
# 1389. Create Target Array in the Given Order
# https://leetcode.com/problems/create-target-array-in-the-given-order/

class Solution:
    def createTargetArray(self, nums: List[int], index: List[int]) -> List[int]:
        ans = []
    
        for i in range(len(nums)):
            ans.insert(index[i], nums[i])
        
        return ans

File: javascript/easy/create_target_array_in_given_order.js
// 1389. Create Target Array in the Given Order
// https://leetcode.com/problems/create-target-array-in-the-given-order/

var createTargetArray = function(nums, index) {
    var answer = []
    
    for (var i = 0; i < nums.length; i++) {
        answer.splice(index[i], 0, nums[i]);
    }
    
    return answer;
};